app/src/app/components/lia-ace-editor/lia-ace-editor.component.spec.ts

```diff
@@ -1,22 +1,14 @@
-import {
-  beforeEach,
-  beforeEachProviders,
-  describe,
-  expect,
-  it,
-  inject,
-} from '@angular/core/testing';
-import { ComponentFixture, TestComponentBuilder } from '@angular/compiler/testing';
+import { addProviders, inject, TestComponentBuilder, ComponentFixture } from '@angular/core/testing';
```

Comment:
Please check [changelog](https://github.com/angular/angular/blob/master/CHANGELOG.md), many testing methods were deprecated and replaced with new once in RC5